tests/user_bonuses/test_creation.py

```diff
@@ -1,9 +1,6 @@
-import datetime
 import re
```

Comment:
I think we are missing:
`test_user_bonus_sync_via_async`
`test_user_bonuses_sync_via_async`
`test_user_bonuses_sync_via_async_retry`